pyproject.toml

```diff
@@ -208,11 +204,13 @@ exclude = [
     "docs/introduction/code/08_main.py",
 ]
 
-[tool.mypy-scrapy]
-ignore_missing_imports = true
-
-[tool.mypy-sortedcollections]
-ignore_missing_imports = true
+[tool.basedpyright]
+pythonVersion = "3.9"
+typeCheckingMode = "standard"
+include = ["src", "tests", "docs"]
+ignore = [
```

Comment:
I tried removing this in another PR and it was ok. Not sure why it is ignored:

https://github.com/apify/crawlee-python/pull/760/commits/2721cc5cc7cce3e9ab478a2c7b1bdae7eb753dfc#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711

Reply:
Let's remove it then, thanks.

Reply:
That was mine :smile: